Gemfile

```diff
@@ -49,6 +51,8 @@ group :development do
 end
 
 group :test do
+  gem 'pry-byebug'
+  gem 'pry-rails'
```

Comment:
Actually it seems that you might be using these during running specs, so they can stay for test. Yes, let's use your solution to get rid of the warning.